### DISCUSSION TITLE
Etudiante Besoin d'aide Elasticsearch

### DISCUSSION THREAD

**Discussion by Amelie80** [Security-Onion-Solutions/securityonion#13821] [Unsupported Versions]
Bonjour, vous allez bien j'espère. 
J'ai déployé Security Onion (Standalone) sur un serveur dédié : Une care réseau que j'utilise pour le management et une autre carte réseau que j'ai connectée sur un port monitor du switch afin sniffer le trafic (j'ai configurer le SPAN). tout fonctionnait correctement , lendemain lorsque je me reconnecter à S.O pour travailler, je remarque dans l'interface GRID qu'il y a une instabilité : 
tantôt il affiche "pending" , tantôt " Fault".
Concernant Elasticsearch : il n'y pas de variation il affiche toujours pending
![WhatsApp Image 2024-10-15 at 1 49 37 PM (1)](https://github.com/user-attachments/assets/bd19792c-96a0-45a0-9eb0-a5345ee17bcf)
![WhatsApp Image 2024-10-15 at 1 49 37 PM](https://github.com/user-attachments/assets/d6f5c903-f26f-4f6e-8489-8bc480995883)
![WhatsApp Image 2024-10-15 at 1 49 37 PM (2)](https://github.com/user-attachments/assets/82be6c30-ce51-4c84-b092-75c79fde79a3)

les ressources du serveur S.O : 
Architecture : Standalone
disk : 230 Go
Ram : 64 Go
J'ai respecté l'ensemble des besoins matériels mentionné dans la documentation pour mon type d'architecture

**Comment by dougburks** [comment]
Was this a fresh installation of version 2.4.110 or an upgrade from a previous version?

Have you made any configuration changes or is this a default installation?

Have you checked the log files for additional clues as shown at https://docs.securityonion.net/en/2.4/help.html and https://docs.securityonion.net/en/2.4/elasticsearch.html#diagnostic-logging?

**Comment by Amelie80** [comment]
C'est une nouvelle installation (6 jours)
En ce qui concerne la configuration  j'ai préféré laisser tous par défaut pour l'instant, donc je n'ai rien configurer si ce n'est que les config de mirroring sur mon switch. 

J'ai pu résoudre le probleme de zeek,
![WhatsApp Image 2024-10-15 at 1 49 37 PM (2)](https://github.com/user-attachments/assets/bfcee3e1-e14e-434a-9e03-2040a1f08974)

Mais depuis, le problème d'elasticsearch persiste, il affiche toujours "pending"
![WhatsApp Image 2024-10-15 at 1 49 37 PM](https://github.com/user-attachments/assets/49cfbec5-0489-4677-b724-c20861f36f96)

Pourtant j'ai 64 Go de RAM et 230 Go espace disk

Honnêtement je ne comprends pas le soucis, j'ai même vérifier les logs je n'ai rien vu d'anormale

**Comment by dougburks** [comment]
Have you checked the logs in /opt/so/log/elasticsearch/securityonion.log?

What is the output of the following?

```
sudo so-elasticsearch-query _cluster/health?pretty
```

```
sudo so-elasticsearch-shards-list | grep UNAS
```

**Comment by Amelie80** [comment]
Voici les sorties que j'obtiens : 
sudo so-elasticsearch-query _cluster/health?pretty : 
![Capture1-21-10-24](https://github.com/user-attachments/assets/2cbb78cc-f787-46bc-ab19-00a7a7ebc1f1)

sudo so-elasticsearch-shards-list | grep UNAS : 
![Capture2-21-10-24](https://github.com/user-attachments/assets/dc8334d3-6ad1-418e-81ad-0631107cd106)

**Reply by dougburks** [reply]
Please see the documentation:
https://docs.securityonion.net/en/2.4/elasticsearch.html#status-pending

**Comment by Amelie80** [comment]
Je vous remercie pour votre assistance @dougburks le problème est résolu.
